src/components/Questionnaire/Submission.jsx

```diff
@@ -2,20 +2,14 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { reduxForm } from 'redux-form';
-import axios from 'axios';
 import InputType from './InputType';
 import ProgressBar from './ProgressBar';
+import { submitApp } from '../../actions/index';
 
 class Submission extends Component {
-  submitApplication(form) { // eslint-disable-line class-methods-use-this
-    console.log(form);
-    axios.post('http://127.0.0.1:8000/applications/', form)
-      .then((response) => {
-        console.log(response);
-      })
-      .catch((error) => {
-        console.log(error);
-      });
+  submitApplication(form) {
+    const { submitApp } = this.props; // eslint-disable-line no-shadow
```

Comment:
Disabling this lint rule on this line makes things ambiguous.

```suggestion
    const { submitApp: diffVarName } = this.props;
```